### PR TITLE
add backend and frontend deployment yaml

### PR DESCRIPTION
**Pull request by asyafalni**
## Type of changes

Backend and Frontend Deployment Config

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?
No deployment config for k8s cluster nodeflux

## What is the new behavior?
Deployment config for backend and frontend

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by sonadztux — APPROVED** [review]

File: deployment/backend.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                secretKeyRef:
+                  name: weber
+                  key: db_password
+            - name: DB_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: weber
+                  key: db_name
+            - name: DB_HOST
+              value: "10.85.160.3"
```
Comment (krismp) [inline deployment/backend.yaml:44]: ini masuk secret aja kaya nya @asyafalni

File: deployment/backend.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            - name: DB_PORT
+              value: "5432"
+            - name: SESSION_EXPIRATION
+              value: "1"
+            - name: URL_ANALYTICS
+              value: https://api.cloud.nodeflux.io/v1/analytics/
+            - name: FACE_ID
+              value: "148197557409939457"
```
Comment (krismp) [inline deployment/backend.yaml:52]: ini fixed value nya?

File: deployment/backend.yaml (hunk 1 above)
Comment (asyafalni) [reply deployment/backend.yaml:44]: boleh @krismp

File: deployment/backend.yaml (hunk 2 above)
Comment (asyafalni) [reply deployment/backend.yaml:52]: iya, sementara itu valuenya fixed disitu..

File: deployment/backend.yaml (hunk 1 above)
Comment (asyafalni) [reply deployment/backend.yaml:44]: paling gw titip, nambahin disecret k8s yak @krismp

Comment (krismp) [reply deployment/backend.yaml]: done gw tambahin

**Review by krismp — APPROVED** [review]

**Review by heronimus — APPROVED** [review]
lgtm gan

**Pull request merged**